prow/README.md

```diff
@@ -117,6 +117,11 @@ deployed then the config will be automatically updated once the PR is merged,
 else you will need to run `make update-plugins`. This does not require 
 redeploying the binaries, and will take effect within a minute.
 
+Note that Github events triggered by the account that is managing the plugins
+are ignored by some plugins. It is prudent to use a different bot account for
+performing merges or rerunning tests, whether the deployment that drives the
+second account is `tide` or the `submit-queue` munger.
```

Comment:
It should work for tide. Does it not?

Reply:
Yeah, it should because tide is not commenting on Github. I didn't go into detail in the doc but I thought that separating the token that has permission to merge from the token used at the entrypoint of the cluster is a good practice. This also allows `tide` to evolve in the future.